### PR TITLE
Use rst instead of md

### PR DESCRIPTION
**Pull request by chrisburr**
Generated using `pandoc -s filename.md -o filename.rst` with `sed -i 's@.md@.rst@g' filename.rst` to fix the hyperlinks.

BEGINRELEASENOTES

CHANGE: Convert docs to rst

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
I think this can be merged. @andresailer would you take care of creating a readthedocs for DIRACOS, and link it from the main DIRAC docs? like done for VMDIRAC.

**Comment by andresailer** [comment]
Yes, I can do that.

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+=======
+
+|pipeline status|
+
+DIRACOS aims at bringing in one archive all the dependencies needed by
+DIRAC to run Agents, Services and clients. It is not intended to perform
+interactive tasks (debugging, file editing, etc). We are aware than some
+“basic” tools won’t work (less, emacs, etc). They won’t be fixed.
+
+-  `Principle <docs/0_concepts.rst#principle>`__
```
Comment (chaen) [inline README.rst:11]: It can go like this, but then we will be able to remove this hand made summary and use the RST generated one :-)

**Comment by chaen** [comment]
@chrisburr can you rebase ? The doc changes you did in the `rpath` PR are now conflicting :)

File: README.rst (hunk 1 above)
Comment (chrisburr) [reply README.rst:11]: If we do that it will no longer appear on GitHub: https://github.com/github/markup/issues/534

Comment (chaen) [reply README.rst:11]: ah :-(

**Pull request merged**